oidc-controller/api/routers/oidc.py

```diff
@@ -78,6 +78,25 @@ async def poll_pres_exch_complete(pid: str, db: Database = Depends(get_db)):
     return {"proof_status": auth_session.proof_status}
 
 
+def gen_deep_link(auth_session: AuthSession) -> str:
+    controller_host = settings.CONTROLLER_URL
+    url_to_message = (
+        controller_host + "/url/pres_exch/" + str(auth_session.pres_exch_id)
+    )
+    suffix = f'_url={base64.b64encode(url_to_message.encode("utf-8")).decode("utf-8")}'
+    if settings.USE_URL_DEEP_LINK:
+        suffix = (
+            f'_url={base64.b64encode(url_to_message.encode("utf-8")).decode("utf-8")}'
+        )
+    else:
+        formated_msg = json.dumps(auth_session.presentation_request_msg)
+        suffix = f'c_i={base64.b64encode(formated_msg.encode("utf-8")).decode("utf-8")}'
```

Comment:
Need to resolve conflicts with changes from this pr: https://github.com/bcgov/vc-authn-oidc/pull/580